plugin/src/sbt-test/sbt-tpolecat/modePerConfiguration/build.sbt

```diff
@@ -22,14 +22,20 @@ Test / tpolecatOptionsMode    := DevMode
 
 TaskKey[Unit]("checkCompileOptions") := {
   val hasFatalWarnings =
-    scalacOptions.value.contains("-Xfatal-warnings")
+    (Compile / scalacOptions).value.contains("-Xfatal-warnings")
```

Comment:
Unscoped settings no longer receive the changes applied by this plugin if we are using configuration filters